src/lib.rs

```diff
@@ -12,6 +12,7 @@
 //! [`debug_assert_matches!`]: macro.debug_assert_matches.html
 
 #![deny(missing_docs)]
+#![cfg_attr(no_std, not(test))]
```

Comment:
You've reversed the arguments here. It should be:

```suggestion
#![cfg_attr(not(test), no_std)]
```

Reply:
Oops, silly me!